analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Tests/VerifierTest.cs

```diff
@@ -76,18 +77,36 @@ public class VerifierTest
 }").Invoking(x => x.Verify()).Should().NotThrow();
 
         [TestMethod]
-        public void Verify_RaiseUnexpectedIssues() =>
-            WithSnippet(
+        public void Verify_RaiseExpectedIssues_VB() =>
+            WithSnippetVB(
+@"Public Class Sample
+    Private A As Integer = 42   ' Noncompliant {{Dummy message}}
+    Private B As Integer = 42   ' Noncompliant
+    Private C As Boolean = True
+End Class").Invoking(x => x.Verify()).Should().NotThrow();
+
+        [TestMethod]
+        public void Verify_RaiseUnexpectedIssues_CS() =>
+            WithSnippetCS(
 @"public class Sample
 {
     private int a = 42;     // FP
     private int b = 42;     // FP
     private bool c = true;
 }").Invoking(x => x.Verify()).Should().Throw<UnexpectedDiagnosticException>().WithMessage("CSharp7: Unexpected primary issue on line 3, span (2,20)-(2,22) with message 'Dummy message'*");
 
+        [TestMethod]
+        public void Verify_RaiseUnexpectedIssues_VB() =>
+            WithSnippetVB(
+@"Public Class Sample
+    Private A As Integer = 42   ' FP
+    Private B As Integer = 42   ' FP
+    Private C As Boolean = True
+End Class").Invoking(x => x.Verify()).Should().Throw<UnexpectedDiagnosticException>().WithMessage("VisualBasic12: Unexpected primary issue on line 2, span (1,27)-(1,29) with message 'Dummy message'*");
+
         [TestMethod]
         public void Verify_MissingExpectedIssues() =>
```

Comment:
I would add also the CS suffix for clarity.

Reply:
It doesn't have VB counterpart. I have the functionality tested for C# and only few tests to make sure it works for VB on main path.